doc/getting-started/building-from-source.rst

```diff
@@ -57,6 +58,20 @@ each with their own way to install development tools:
         $ sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 60 \
                              --slave /usr/bin/g++ g++ /usr/bin/g++-7
 
+  .. note::
+     You need a newer version of ``binutils`` on Ubuntu 16.04. The latest
```

Comment:
This would be better written as:
```
ACRN development requires ``binutils`` version 2.27 (or higher).
You can verify your version of ``binutils`` with the command 
``apt-show-versions binutils``. While Ubuntu 18.04 has a new version
of ``binutils`` the default version on Ubuntu 16.04 needs updating (see
issue `#1133 <https://github.com/projectacrn/acrn-hypervisor/issues/1133>`_ ).
```

Reading through this whole doc though, I don't see us mentioning anything other than Ubuntu 16.04.  Should this doc be updated to mention 18.04?

Reply:
Possibly, I haven't tried building on Ubuntu 18.04 for a while and was running out of time after debugging and updating the 16.04 instructions. I decided it was better to at least update this portion for now rather than wait to have a more complete update.

It's been a while since I've tried the various distros we list to be honest, it could be that various sections need updating. Fedora 29 for example needs a patch (see #2457) but that is also not documented anywhere yet.

Reply:
> This would be better written as:

I have updated the PR with the text you suggested.

> You can verify your version of ``binutils`` with the command 
> ``apt-show-versions binutils``

Except this part as I do not have `apt-show-versions` available in my container for example, so instead of adding the tool, I used `apt show binutils` which does the same thing out of the box.